pkg/healthcheck/healthcheck_output.go

```diff
@@ -48,20 +52,142 @@ func (cr CheckResults) RunChecks(observer CheckObserver) bool {
 	return success
 }
 
+// PrintCoreChecksHeader writes the core checks header if the output format is not
+// JSON. Used by the check cmd package.
+func PrintCoreChecksHeader(wout io.Writer, output string) {
+	if output != JSONOutput {
```

Comment:
rather than passing in the `output` parameter and checking it inside the function, I think this check should be done at the call site.  i.e.

```
if output != JSONOutput {
    PrintCoreChecksHeader(wout)
}
```